extensions/blocks/story/player/style.scss

```diff
@@ -8,6 +8,7 @@
 	width: 180px;
 	margin-left: auto;
 	margin-right: auto;
+	margin-bottom: 24px;
```

Comment:
We're using `20px` on the block-editor (for `wp-story-with-controls`) maybe we could unify the 2?

Reply:
Makes sense - I dropped it from `wp-story-with-controls` in 8717e08 so it'll just keep the `24px`. Let me know if that's no what you had in mind.

Reply:
Yep looks good now!